### PR TITLE
Add output for Python code

### PR DESCRIPTION
**Pull request by mvantellingen**
The Python code generated uses Marshmallow for serializing and deserializing JSON.

The generated code is available at https://github.com/labd/commercetools-python-sdk

### CODE REVIEW

**Review by jenschude — APPROVED** [review]

**Comment by mvantellingen** [comment]
Thanks. I want to clean up some more code before merging it. I'll update the PR when ready

**Comment by mvantellingen** [comment]
Ok I'm fairly happy with the state of the code for now. I think there are still lot's of things that can be done in a better, more perhaps kotlinic 😄  way, but this works

File: languages/python/src/main/kotlin/io/vrap/codegen/languages/python/PyStringExtension.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    val name = StringCaseFormat.LOWER_UNDERSCORE_CASE.apply(this.replace(".", "_"))
+    var keywords = arrayOf(
+        "False", "None", "True", "and", "as", "assert", "async", "await",
+        "break", "class", "continue", "def", "del", "elif", "else", "except",
+        "finally", "for", "from", "global", "if", "import", "in", "is",
+        "lambda", "nonlocal", "not", "or", "pass", "raise", "return", "try",
+        "while", "with", "yield"
+    )
+    if (keywords.contains(name)) {
+        return name + "_"
```
Comment (jenschude) [inline languages/python/src/main/kotlin/io/vrap/codegen/languages/python/PyStringExtension.kt:47]: oh btw in java we prefixed keywords with underscore. Don't know how it's handled for python

Comment (mvantellingen) [reply languages/python/src/main/kotlin/io/vrap/codegen/languages/python/PyStringExtension.kt:47]: So in the Python world there are no true private variables so instead private variables are prefixed with an underscore. So when we prefix with an underscore people (and most likely linting tools) will think it's private.

Adding a _ as suffix is often used to differentiate with keywords.

Comment (jenschude) [reply languages/python/src/main/kotlin/io/vrap/codegen/languages/python/PyStringExtension.kt:47]: okay 👍

**Pull request merged**